### PR TITLE
Toggle adding mode, fix marker loading issues, change navbar colour

### PR DESCRIPTION
**Pull request by peterjcole**
- Add a button which will toggle 'add mode'
- Fix for marker images not always loading
- Change marker for adding a bin
- Change navbar colour
- Make the selection for bin type visible (including the default selection of 'Mixed')
- Change the selection values to make the POST request submit the correct bin type names ('Mixed' rather than 'mixed')

### CODE REVIEW

File: public/map/mapbox.js [hunk 1]
```diff
@@ -13,54 +14,94 @@ const geoLocation = new mapboxgl.GeolocateControl({
   showUserLocation: true
 });
 
+map.on('styleimagemissing', async function(e) {
```
Comment (AnnaWeakclaw) [inline public/map/mapbox.js]: not sure what this is doing :(

File: public/map/mapbox.js [hunk 2]
```diff
@@ -13,71 +14,142 @@ const geoLocation = new mapboxgl.GeolocateControl({
...
+  var id = e.id
+  console.log(id)
```
Comment (gps2601) [inline public/map/mapbox.js]: do we need these console logs still?

File: resources/mapMarkers/download.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg viewBox='0 0 20 20' xmlns='http://www.w3.org/2000/svg' fill='#33b5e5'> <path d='M 10,4 C 9,4 9,5 9,5 L 9,5.1 C 7.0357113,5.5006048 5.5006048,7.0357113 5.1,9 L 5,9 c 0,0 -1,0 -1,1 0,1 1,1 1,1 l 0.1,0 c 0.4006048,1.964289 1.9357113,3.499395 3.9,3.9 L 9,15 c 0,0 0,1 1,1 1,0 1,-1 1,-1 l 0,-0.1 c 1.964289,-0.400605 3.499395,-1.935711 3.9,-3.9 l 0.1,0 c 0,0 1,0 1,-1 C 16,9 15,9 15,9 L 14.9,9 C 14.499395,7.0357113 12.964289,5.5006048 11,5.1 L 11,5 c 0,0 0,-1 -1,-1 z m 0,2.5 c 1.932997,0 3.5,1.5670034 3.5,3.5 0,1.932997 -1.567003,3.5 -3.5,3.5 C 8.0670034,13.5 6.5,11.932997 6.5,10 6.5,8.0670034 8.0670034,6.5 10,6.5 Z'/> </svg>
```
Comment (gps2601) [inline resources/mapMarkers/download.svg]: 😡

**Review by gps2601 — COMMENTED** [review]
Great work! Looks really good 🗺🗑

**Review by gps2601 — DISMISSED** [review]

**Review by AnnaWeakclaw — DISMISSED** [review]
Great job! 💯

**Review by AnnaWeakclaw — APPROVED** [review]
💯

**Pull request merged**